### PR TITLE
Polars: evenly allocate budget amongst queries, by default

### LINKED ISSUE #1757: Polars: evenly allocate privacy budget amongst DP expressions
When two queries have significantly different sensitivities, the query with less sensitivity will get less noise relative to the other query.

In extreme cases, like when estimating quantiles where alpha breaks into a large fraction (1/3), the sensitivity is very large, and the scale parameter works out to be proportionately large, starving the privacy budget of any other queries.

### PR DESCRIPTION
**Pull request by Shoeboxam**
Closes #1757

### CODE REVIEW

**Comment by Shoeboxam** [comment]
* **#1825** <a href="https://app.graphite.dev/github/pr/opendp/opendp/1825?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Shoeboxam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python/test/test_polars.py [hunk 1]
```diff
@@ -369,7 +369,7 @@ def test_polars_describe():
             "column": ["len", "A"],
             "aggregate": ["Len", "Sum"],
             "distribution": ["Integer Laplace", "Integer Laplace"],
-            "scale": [8.0, 8.0],
+            "scale": [4.0, 12.0],
```
Comment (mccalluc) [inline python/test/test_polars.py:372]: Can you add a comment explain the 1/3 ratio? I see this in the original issue, and I can imagine that sum is more sensitive than len, but I don't immediately see where that ratio is coming from.

**Review by mccalluc — COMMENTED** [review]
Add a comment to explain the ratio in the python test? Thanks.

File: python/test/test_polars.py [hunk 2]
```diff
@@ -366,25 +366,32 @@ def test_polars_describe():
...
         }
     )
 
+    summer = pl.col("A").fill_null(2).dp.sum((0, 3))
+
     query = (
         context.query()
         .group_by("B")
-        .agg(pl.len().dp.noise(), pl.col("A").fill_null(2).dp.sum((0, 3)))
+        .agg(pl.len().dp.noise(), summer, summer.alias("B"))
```
Comment (mccalluc) [inline python/test/test_polars.py:385]: I'm not sure how the introduction of the aliased column strengthens the test.

**Review by mccalluc — APPROVED** [review]
The explanation is helpful: Thanks! The alias column is just to make sure it still comes through when it's a little more complicated? Approved.

File: python/test/test_polars.py (hunk 2 above)
Comment (Shoeboxam) [reply python/test/test_polars.py:385]: It prevents the second sum from shadowing "A". Polars will otherwise reject it.

**Pull request merged**